package.json

```diff
@@ -25,7 +25,7 @@
     "@sendgrid/mail": "^6.4.0",
     "bignumber.js": "^8.0.1",
     "dotenv": "^6.2.0",
-    "eslint": "5.12.0",
+    "eslint": "5.16.0",
```

Comment:
Was this on purpose? (Totally cool if it was, just wanted to double check)

Reply:
Yeah `npm start` was complaining about a version mismatch without this update.